platform/node/README.md

```diff
@@ -29,13 +29,25 @@ npm run test-suite
 ## Rendering a map tile
 
 ```js
+var fs = require('fs');
+var path = require('path');
 var mbgl = require('mapbox-gl-native');
 var sharp = require('sharp');
-var map = new mbgl.Map({ request: function() {} });
+
+var options = {
+  request: function(req, callback) {
+    fs.readFile(path.join(__dirname, 'test', req.url), function(err, data) {
+      callback(err, { data: data });
+    });
+  },
+  ratio: 1
+};
+
+var map = new mbgl.Map(options);
 
 map.load(require('./test/fixtures/style.json'));
 
-map.render({}, function(err, buffer) {
+map.render({zoom: 0}, function(err, buffer) {
```

Comment:
This shouldn't be necessary, as a zoom level of `0` is the default [when no options are specified](https://github.com/mapbox/mapbox-gl-native/blob/08d0a9a0e2e771b87018bcb3d4d124c52fed1689/platform/node/src/node_map.cpp#L24).